test/units/module_utils/common/text/formatters/test_lenient_lowercase.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019, Andrew Klychkov @Andersson007 <aaklychkov@mail.ru>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils.common.text.formatters import lenient_lowercase
+
+
+INPUT_LIST = [
+    u'HELLO',
+    u'Ёлка',
+    u'cafÉ',
+    b'HELLO',
+    1,
+    {1: 'Dict'},
+    True,
+    [1],
```

Comment:
Add a float test case.
```suggestion
    [1],
    3.14159,
```